magik-checks/src/main/java/nl/ramsolutions/sw/magik/checks/checks/ForbiddenInheritanceCheck.java

```diff
@@ -13,21 +13,26 @@
 import nl.ramsolutions.sw.magik.checks.DisabledByDefault;
 import nl.ramsolutions.sw.magik.checks.MagikCheck;
 import nl.ramsolutions.sw.magik.parser.TypeStringParser;
+import org.sonar.check.Rule;
 import org.sonar.check.RuleProperty;
 
 /**
  * Check if forbidden inheritance is used.
  */
 @DisabledByDefault
+@Rule(key = ForbiddenInheritanceCheck.CHECK_KEY)
 public class ForbiddenInheritanceCheck extends MagikCheck {
 
+    @SuppressWarnings("checkstyle:JavadocVariable")
+    public static final String CHECK_KEY = "ForbiddenInheritance";
+
     private static final String DEFAULT_FORBIDDEN_PARENTS = "";
 
     /**
      * Forbidden parents to inhertit from, separated by ','.
      */
     @RuleProperty(
-        key = "fobidden parents",
+        key = "forbidden parent(s)",
```

Comment:
I think this will give problems when configuring the check via `magik-lint.properties`. Lets keep it `forbidden parents` (include the missing `r`, undo the `(s)`).